### PR TITLE
peer code review JS best practices

### PR DESCRIPTION
**Pull request by joyapisi**
REquest to review my `To Do List` project

### CODE REVIEW

File: src/modules/add-delete-update.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+function clearDoneTasks(todoTasks, task) {
+  const index = todoTasks.findIndex((item) => item.index === task.index);
+  todoTasks.splice(index, 1);
+  todoTasks.forEach((task, index) => {
+    task.index = todoTasks.length - index;
+  });
+  return todoTasks;
+}
```
Comment (lilskyex0x) [inline src/modules/add-delete-update.js:8]: - [ ] I would suggest that to use arrow functions for the functions😄. It is not that mandatory but if you wanna follow JavaScript best practices, it's better to use the ES6 feature in your code. Kindly fix that one would be awesome😊

Comment (lilskyex0x) [inline src/modules/add-delete-update.js:8]: - [x] [optional] I would suggest that to use arrow functions for the functions😄. It is not that mandatory but if you wanna follow JavaScript best practices, it's better to use the ES6 feature in your code. Kindly fix that one would be awesome😊

**Review by lilskyex0x — APPROVED** [review]
Hi @joyapisi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/add-delete-update.js (hunk 1 above)
Comment (joyapisi) [reply src/modules/add-delete-update.js:8]: Thank you! I will work on that.

> * [x]  [optional] I would suggest that to use arrow functions for the functions😄. It is not that mandatory but if you wanna follow JavaScript best practices, it's better to use the ES6 feature in your code. Kindly fix that one would be awesome😊

**Pull request merged**